modules/claims_api/spec/requests/v2/rswag_veteran_identifier_request_spec.rb

```diff
@@ -0,0 +1,140 @@
+# frozen_string_literal: true
+
+require 'swagger_helper'
+require 'rails_helper'
+require_relative '../../support/swagger_shared_components'
+
+describe 'Veteran Identifier', swagger_doc: 'v2/swagger.json' do # rubocop:disable RSpec/DescribeClass
```

Comment:
shouldnt this swagger_doc value (but v1 instead) be added to all the existing v1 rswag spec files?

Reply:
We don't have to because of [this](https://github.com/rswag/rswag#supporting-multiple-versions-of-api).
> "_By default, the paths, operations and responses defined in your spec files will be associated with the **first** Swagger document in swagger_helper.rb._"

We can explicitly define that `swagger_doc` param on v1 classes if we want.  We just don't "have" to.  🤷

Reply:
gotcha. its probably something we'll need to do if another team decides to start using rswag i feel like?

Reply:
Fair enough.  
[Updated here.](https://github.com/department-of-veterans-affairs/vets-api/pull/7195/commits/57b3a2c69b68f1426f22ae630273eb3289690439)